docs-ref-conceptual/includes/cli-install-linux-apt.md

```diff
@@ -40,25 +40,27 @@ If you prefer a step-by-step installation process, complete the following steps
 
     ```bash
     sudo apt-get update
-    sudo apt-get install ca-certificates curl apt-transport-https lsb-release gnupg
+    sudo apt-get install apt-transport-https ca-certificates curl gnupg2 lsb-release
     ```
 
 2. Download and install the Microsoft signing key:
 
     ```bash
     sudo mkdir -p /etc/apt/keyrings
-    curl -sLS https://packages.microsoft.com/keys/microsoft.asc |
-        gpg --dearmor |
-        sudo tee /etc/apt/keyrings/microsoft.gpg > /dev/null
-    sudo chmod go+r /etc/apt/keyrings/microsoft.gpg
```

Comment:
After removing this line, will group/other still have read permission for `/etc/apt/keyrings/microsoft.gpg`?

Reply:
yes, unless the umask restricts it. otoh, don't think anyone except root needs to read it, but i'll add the `chmod`